### ISSUE TITLE
Support UUID in PostgreSQL connector

### ISSUE THREAD

**Issue by pgagnon** [prestosql/presto#647]
UUID columns in Postgresql tables are silently dropped in Presto. This was previously reported in  prestodb/presto#10951 and @findepi gave implementation directions.

I will probably submit a PR for this in the near-term since this is a feature that I need. I'd like to gather some opinions though on either if UUID types should be represented as a binary(16) or a char(36).

**Comment by ebyhr** [comment]
As you may already know, cassandra connector maps UUID to varchar(36). I feel same mapping looks better if there's no special reason to use binary type.

**Comment by ebyhr** [comment]
Just confirmed current unsupported types in postgresql connector.
```
bit varying
cidr
circle
inet
interval
line
lseg
macaddr
path
pg_lsn
point
polygon
tsquery
tsvector
txid_snapshot
uuid
xml
```

**Comment by findepi** [comment]
@ebyhr thanks for the list.
To steer the discussion -- fixing one (or small few) per PR will be probably most efficient,
unless there is some strong binding pattern between the types (eg. `tsquery`, `tsvector`, `tsrange`).
A PR that tries to fix many types at ones will be hard to review and is likely to have more iterations.

Also, let's start with types that matter most, i.e. are needed most broadly. UUID is a likely winner.
@pgagnon are you going to UUID in Postgres?
Doing it the same way it's done in Cassandra is reasonable to me.

**Comment by pgagnon** [comment]
@findepi Yes, I will PR UUIDs perhaps today.

**Comment by electrum** [comment]
I think we should add a UUID type to Presto. The type can use the int128 block internally but render in the standard textual format. 

For now, we only need the type. We might also add a few basic functions like generating different UUID variants, but that can be done later.

**Comment by pgagnon** [comment]
Just to keep everyone in the look I have spoken with @Praveen2112 and he is planning on PR'ing UUIDs as presto types. This will be on hold for a little bit in order to sync with his work.

**Comment by martint** [comment]
@pgagnon, UUID type support has been merged.

**Comment by z0le** [comment]
Hi @Praveen2112 I can work on the UUID type if you are not planning on doing this soon. Let me know.

**Comment by Praveen2112** [comment]
@z0le Please check with @pgagnon I guess he is working on it.

**Comment by pgagnon** [comment]
@z0le I have done some preliminary work on this but I have not had as much time as I wanted. Let me know if you want to take this over and I'll yield to you, otherwise I'll try and get this done by early next week.

**Comment by z0le** [comment]
@pgagnon early next week is fine, but if you still don't have time I can do this, let me know.

**Comment by pgagnon** [comment]
@z0le I have implemented this in #1011.
